content/smart.md

```diff
@@ -0,0 +1,126 @@
+---
+title: SMART
+layout: smart
+---
+
+# SMART<sup>®</sup> #
+------------------------------------------------------------------------
+* TOC
+{:toc}
+
+## General ##
+
+### Introduction ###
+The SMART<sup>®</sup> (Substitutable Medical Apps and Reusable Technology) platform defines a specification for an electronic health record (EHR) to safely and securely open other applications with context. These SMART applications are commonly web applications but may also be native mobile applications and that use HL7® FHIR® [standard][https://www.hl7.org/fhir/] to read and write data from the EHR. With SMART, Cerner can embed a SMART app in the EHR. Cerner believes that SMART applications will be a major user of FHIR resources. We will also support FHIR access through mobile SMART applications, as those specifications emerge from the SMART [web site][http://smartplatforms.org/].
+
+### SaaS Model ###
+
+Cerner expects the majority of SMART apps to be provided to clients in a SaaS (software as a service) model. That is, the SMART app is hosted or managed by the SMART developer or provider. In this case, clients do not need to install any code or package in order to implement a particular SMART app.
+
+### Authorization Model ###
+
+At a high level, provider facing applications must receive the following authorizations:
+
+1. Cerner must validate and register every SMART app. As part of this process, Cerner validates the usage of FHIR resources and operations (for example, READ and WRITE).  Cerner will not validate the usage of FHIR resources for direct to consumer apps.
+2. Each client decides whether they want to allow a SMART app to run on a domain-by-domain basis.
+3. Each client decides which users and roles can access the SMART app.
+
+To learn more about general SMART on FHIR app authorization process, see the SMART [App Authorization Guide][http://docs.smarthealthit.org/authorization/].  For implementation information regarding Cerner's Authorization server, see the [Authorization page][http://fhir.cerner.com/authorization/] at fhir.cerner.com.
+
+### Access Points ###
+
+Currently, end users can open SMART apps in the following ways:
+
+- Providers can access SMART apps from the *Cerner Millennium PowerChart* TOC (table of contents, a column of options that open different PowerChart components) and the Organizer view
```

Comment:
table of contents, a column of options that **opens** different PowerChart components.